package.nls.json

```diff
@@ -191,6 +191,8 @@
 	"command.review.requestChanges.title": "Request Changes",
 	"command.review.approveOnDotCom.title": "Approve on github.com",
 	"command.review.requestChangesOnDotCom.title": "Request changes on github.com",
+	"command.review.createSuggestionsFromChanges.title": "Create Pull Request Suggestions",
+	"command.review.createSuggestionsFromChange.title": "Convert to Pull Request Suggestion",
```

Comment:
Duplicate line?

Reply:
Accidentally included. The next change for the feature needs that line.